plugins/source/digitalocean/codegen/resources.go

```diff
@@ -0,0 +1,398 @@
+			},
+		},
+		SkipFields: []string{"URN"},
+	},
+	{
+		Service:   "registries",
+		Struct:    &godo.Registry{},
+		Relations: []string{"Repositories()"},
+		ExtraColumns: []codegen.ColumnDefinition{
+			{
+				Name:     "name",
+				Type:     schema.TypeString,
+				Resolver: `schema.PathResolver("Name")`,
+				Options:  schema.ColumnCreationOptions{PrimaryKey: true},
+			},
+		},
+		SkipFields: []string{"Name"},
+	},
+	{
+		Service:    "registries",
+		SubService: "repositories",
+		Struct:     &godo.Repository{},
+		ExtraColumns: []codegen.ColumnDefinition{
+			{
+				Name:     "name",
+				Type:     schema.TypeString,
+				Resolver: `schema.PathResolver("Name")`,
+				Options:  schema.ColumnCreationOptions{PrimaryKey: true},
+			},
+		},
+		SkipFields: []string{"Name"},
+	},
+	{
+		Service: "sizes",
+		Struct:  godo.Size{},
+	},
+	{
+		Service: "snapshots",
+		Struct:  godo.Snapshot{},
+		ExtraColumns: []codegen.ColumnDefinition{
+			{
+				Name:     "id",
+				Type:     schema.TypeString,
+				Resolver: `schema.PathResolver("ID")`,
+				Options:  schema.ColumnCreationOptions{PrimaryKey: true},
+			},
+		},
+		SkipFields: []string{"ID"},
+	},
+	{
+		Service:      "spaces",
+		Struct:       spaces.WrappedBucket{},
+		PostResolver: "resolveSpaceAttributes",
+		Multiplex:    "client.SpacesRegionMultiplex",
+		Imports:      []string{"github.com/cloudquery/cloudquery/plugins/source/digitalocean/client"},
+		Relations:    []string{"Cors()"},
+		ExtraColumns: []codegen.ColumnDefinition{
+			{
+				Name:     "acls",
+				Type:     schema.TypeJSON,
+				Resolver: `schema.PathResolver("ACLs")`,
+			},
+		},
+		SkipFields: []string{"ACLs"},
+	},
+	{
+		Service:    "spaces",
+		SubService: "cors",
+		Struct:     types.CORSRule{},
+		ExtraColumns: []codegen.ColumnDefinition{
+			{
+				Name:     "id",
+				Type:     schema.TypeString,
+				Resolver: `schema.PathResolver("ID")`,
+				Options:  schema.ColumnCreationOptions{PrimaryKey: true},
+			},
+		},
+		SkipFields: []string{"ID"},
+	},
+	{
+		Service:    "storage",
+		SubService: "volumes",
+		Struct:     godo.Volume{},
+		ExtraColumns: []codegen.ColumnDefinition{
+			{
+				Name:     "id",
+				Type:     schema.TypeString,
+				Resolver: `schema.PathResolver("ID")`,
+				Options:  schema.ColumnCreationOptions{PrimaryKey: true},
+			},
+			{
+				Name:     "droplet_ids",
+				Type:     schema.TypeIntArray,
+				Resolver: `schema.PathResolver("DropletIDs")`,
+			},
+		},
+		SkipFields: []string{"DropletIDs", "ID"},
+	},
+	{
+		Service:   "vpcs",
+		Struct:    &godo.VPC{},
+		Relations: []string{"Members()"},
+		ExtraColumns: []codegen.ColumnDefinition{
+			{
+				Name:     "id",
+				Type:     schema.TypeString,
+				Resolver: `schema.PathResolver("ID")`,
+				Options:  schema.ColumnCreationOptions{PrimaryKey: true},
+			},
+		},
+		SkipFields: []string{"ID"},
+	},
+	{
+		Service:    "vpcs",
+		SubService: "members",
+		Struct:     &godo.VPCMember{},
```

Comment:
I noticed that the generated table name here becomes `digitalocean_vpcs_members` - it would be great if it could be `digitalocean_vpc_members`. Would that be possible? 😬